### PR TITLE
Run Jakarta Authorization test via mvn on current build

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: appserver/tests/tck/pom.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.glassfish.main.tests</groupId>
+        <artifactId>tests</artifactId>
```
Comment (dmatej) [inline appserver/tests/tck/pom.xml:25]: Probably this module should be added also to it's parent, or at least to the `<id>set-version-id</id>` profile, so it's version could be updated automatically in the release process.

**Review by dmatej — CHANGES_REQUESTED** [review]

File: appserver/tests/tck/authorization/pom.xml [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>Grab-Glassfish</id>
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply appserver/tests/tck/authorization/pom.xml:48]: Grab is done to mirror the same name used by the CI job. It doesn't matter that much to me, but that's where the name comes from.

File: appserver/tests/tck/authorization/pom.xml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    </parent>
+
+    <artifactId>glassfish-external-tck-authorization</artifactId>
+    <packaging>pom</packaging>
+
+    <name>GlassFish - External - TCK - Authorization</name>
+
+    <properties>
+        <ant.home>${project.build.directory}/ant</ant.home>
+        <TCK_HOME>${project.build.directory}/tck</TCK_HOME>
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply appserver/tests/tck/authorization/pom.xml:36]: +1

File: appserver/tests/tck/authorization/pom.xml [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        <glassfish.home>${TCK_HOME}/glassfish6</glassfish.home>
+        <test.home>${TCK_HOME}/src/com/sun/ts/tests</test.home>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply appserver/tests/tck/authorization/pom.xml:44]: +1

File: appserver/tests/tck/authorization/pom.xml [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>Grab-Glassfish</id>
+                        <phase>initialize</phase>
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply appserver/tests/tck/authorization/pom.xml:49]: +1

File: appserver/tests/tck/authorization/pom.xml [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.0.0</version>
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply appserver/tests/tck/authorization/pom.xml:72]: +1

File: appserver/tests/tck/authorization/pom.xml [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+                                <artifactId>ant</artifactId>
+                            </exclusion>
+                        </exclusions>
+                    </dependency>
+                </dependencies>
+                
+                <executions>
+                    <execution>
+                        <id>Grab-TCK-ANT-Configure-ts</id>
+                        <phase>validate</phase>
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply appserver/tests/tck/authorization/pom.xml:95]: +1

**Review by jGauravGupta — APPROVED** [review]

File: appserver/tests/tck/pom.xml (hunk 1 above)
Comment (arjantijms) [reply appserver/tests/tck/pom.xml:25]: Yes, I was thinking about that. Let's add it to its parent now. We can add profiles to skip running too many tests during casual builds later.

**Review by dmatej — APPROVED** [review]

**Pull request merged**